server/controllers/auth-controller.js

```diff
@@ -33,13 +45,31 @@ class UsersController {
         try {
           const { dataValues: user } = await User.create({
             ...body,
-            hash: body.password
+            hash: body.password,
           });
           const { id, username } = user;
-          const token = generateToken({ id, username });
+          const [token, verificationToken] = await authTokens({
```

Comment:
What's the purpose of the two tokens here?

Reply:
one has a shorter lifespan than the other..... they both are also configurable via external config vars

Reply:
I am quite aware of the difference between them. I'm asking what the purpose of the (authentication) token is here.

Reply:
The authentication token would be required to validate user access to protected routes.